apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/MetadataItem/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { ReactNode } from 'react'
+
+import { ProviderLogDto } from '@latitude-data/core/browser'
+import { Icon, Skeleton, Text, Tooltip } from '@latitude-data/web-ui'
+import { FinishReason } from 'ai'
+
+type MetadataItemProps = {
+  stacked?: boolean
+  label: string
+  value?: string
+  loading?: boolean
+  children?: ReactNode
+}
+
+export function MetadataItem({
+  label,
+  stacked = false,
+  value,
+  loading,
+  children,
+}: MetadataItemProps) {
+  const className = stacked
+    ? 'flex-flex-col gap-2'
+    : 'flex flex-row justify-between items-center gap-2'
+  return (
+    <div className={className}>
+      <Text.H5M color='foreground'>{label}</Text.H5M>
+      <div>
+        {loading ? (
+          <Skeleton className='w-12 h-4 bg-muted-foreground/10' />
```

Comment:
the background class shouldn't be needed and you have the height prop for setting height